common/djangoapps/course_modes/views.py

```diff
@@ -175,6 +176,8 @@ def get(self, request, course_id, error=None):
                             }
 
         context = {
+            "default_currency": settings.PAID_COURSE_REGISTRATION_CURRENCY[0],
+            "default_currency_symbol": settings.PAID_COURSE_REGISTRATION_CURRENCY[1],
```

Comment:
nit: PAID_COURSE_REGISTRATION_CURRENCY.name and PAID_COURSE_REGISTRATION_CURRENCY.symbol might be a better option.

Reply:
I didn't change it as edX is already using this variable as a list. I am just using those settings.